src/main/thrift/parquet.thrift

```diff
@@ -197,18 +197,78 @@ enum FieldRepetitionType {
   REPEATED = 2;
 }
 
+/** Identifier for the sort order used to produce min and max values. */
+enum Order {
+  /**
+   * The signed ordering is the order produced by comparing single primitive
+   * values with a signed comparator, or the lexicographic ordering produced by
+   * comparing each byte of a binary or fixed using a signed comparator.
+   *
+   * (A signed comparator uses the most-significant bit as a sign bit; an
+   * unsigned comparator uses the most-significant bit as part of the value's
+   * magnitude. Note that unsigned comparison is not defined for floating
+   * point values.)
+   */
+  SIGNED = 0;
+
+  /**
+   * The unsigned ordering is produced by comparing single primitive values
+   * with an unsigned comparison, or the lexicographic ordering produced by
+   * comparing eacy byte of a binary or fixed using an unsigned comparator.
+   *
+   * (A signed comparator uses the most-significant bit as a sign bit; an
+   * unsigned comparator uses the most-significant bit as part of the value's
+   * magnitude. Note that unsigned comparison is not defined for floating
+   * point values.)
+   */
+  UNSIGNED = 1;
+
+  /**
+   * Identifiers for custom orderings, to be defined using CustomOrder entries
+   * in the file footer.
+   */
+  CUSTOM1 = 2;
+  CUSTOM2 = 3;
+  CUSTOM3 = 4;
+  CUSTOM4 = 5;
+  CUSTOM5 = 6;
+  CUSTOM6 = 7;
+}
+
+/** A mapping from Order place-holders to a string descriptor */
+struct CustomOrder {
+  /** An Order place-holder value used in this file */
+  1: optional Order order;
+
+  /**
+   * A string that identifies the order denoted by the place-holder order enum
+   * symbol. Ordering descriptors should be well-known identifiers, such as the
+   * locale keywords described in ICU 54.
+   *
+   * See http://userguide.icu-project.org/collation/api#TOC-Instantiating-the-Predefined-Collators
+   */
+  2: optional string descriptor;
+}
+
 /**
  * Statistics per row group and per page
  * All fields are optional.
  */
 struct Statistics {
-   /** min and max value of the column, encoded in PLAIN encoding */
-   1: optional binary max;
-   2: optional binary min;
-   /** count of null value in the column */
-   3: optional i64 null_count;
-   /** count of distinct values occurring */
-   4: optional i64 distinct_count;
+  /** min and max value of the column, encoded in PLAIN encoding */
+  1: optional binary max;
+  2: optional binary min;
+  /** count of null value in the column */
+  3: optional i64 null_count;
+  /** count of distinct values occurring */
+  4: optional i64 distinct_count;
+  /**
+   * Order used to produce min and max values.
+   *
+   * If this field is missing, the order must be the signed ordering for
+   * backward-compatibility.
+   */
+  5: optional Order order;
```

Comment:
Yeah I don't have a concrete use-case for it right now but I think it might be possible that users want to support multiple kinds of queries over their data (one with locale = x, one with locale = y)? Don't think we need to handle this right now though.